indra/databases/pubchem_client.py

```diff
@@ -74,3 +74,28 @@ def get_preferred_compound_ids(pubchem_cid):
                 pref_ids |= set(pref_cpd['Value']['Number'])
     pref_ids = sorted([str(pid) for pid in pref_ids])
     return pref_ids
+
+
+def get_pmids(pubchem_cid):
```

Comment:
```suggestion
def get_pmids(pubchem_cid: str) -> List[str]:
```

good time to check that you're explicitly handling integer pmids

Reply:
Thanks Charlie! will make the changes now